content/docs/command-reference/exp/rename.md

```diff
@@ -0,0 +1,80 @@
+# exp rename
+
+Rename a specific experiment from the <abbr>project</abbr>.
+
+## Synopsis
+
+```usage
+usage: dvc exp rename [-h] [-q | -v] [--rev <commit>]
+                      [-g <git_remote>]
+                      [experiment] [name]
+
+positional arguments:
+   experiment    Experiments to remove.
+   name          Name of new experiment.
+```
+
+## Description
+
+Renames one experiment, indicated by name (see `dvc exp run`) or rev.
+
+## Options
+
+- `--rev <commit>` - rename experiments derived from the specified `<commit>` as
```

Comment:
```suggestion
- `--rev <commit>` - rename all experiments derived from the specified `<commit>` as
```
Does that make sense? multiple experiments with the same name 🤔

Reply:
I believe it should only rename one experiment, but experiment names are only unique by `rev`, so you may need to specify the `rev` to rename an experiment from any commit other than the current HEAD. @BradyJ27 If that's the case, can we clarify the default behavior?

Reply:
Removed `--rev` flag in general.